File: Naftan.Maintenance.JS/src/views/measureUnit.js
webix.protoUI({

    name: "view_measureunit",
    collection: "measureUnit",
    editColumn: "name",

    $init: function (config) {

        webix.extend(config.rows[1], {
            columns: [
                webix.column("id"),
                { id: 'name', header: ['Наименование', { content: "textFilter" }], sort: "string", editor: "text", fillspace: true },
                { id: 'designation', header: ['Обозначение', { content: "textFilter" }], sort: "string", editor: "text", fillspace: true },
                { id: 'description', header: ['Примечание', { content: "textFilter" }], sort: "string", editor: "text", fillspace: true },
                webix.column("trash")
            ]
        }, true);

    }

}, webix.ui.ui_editdatatable);

File: Naftan.Maintenance.Domain/MeasureUnit.cs
using Naftan.Common.Domain;

namespace Naftan.Maintenance.Domain
{
    /// <summary>
    /// Единица измерения
    /// </summary>
    public class MeasureUnit:IEntity
    {
        public int Id { get; set; }
        /// <summary>
        /// Наименование
        /// </summary>
       public string Name { get; set; }
        /// <summary>
        /// Обозначение
        /// </summary>
        public string Designation { get; set; }
        /// <summary>
        /// Описание
        /// </summary>
        public string Description { get; set; }
    }
}